std/string.d

```diff
@@ -5288,6 +5288,7 @@ if (isSomeString!S)
 }
 
 
+/*
 @safe pure @nogc unittest
```

Comment:
Wasn't it possible to mark the unittest as deprecated, which allows testing deprecated code? Or was I dreaming?